templates/Form/fields.html.twig

```diff
@@ -110,7 +110,7 @@
 
     {% if allow_add %}
       <button type="button" class="btn btn-success btn-sm" data-role="collection-add-button">
-        <i class="fas fa-plus me-2"></i> {{ 'forms.buttons.addItem'|trans|ucfirst }}
+        <i class="fas fa-plus me-2"></i> {{ form.vars.add_button_label|trans|ucfirst }}
```

Comment:
Wil die lijn in CollectionTypeExtension niet juist zeggen dat ge dat kunt aanroepen als `add_button_label` zonder die form.vars?

Reply:
fixed 78a87a7551da0216bd8d2f8711245bca0442e88c